src/Symfony/Component/HttpKernel/EventListener/ProfilerListener.php

```diff
@@ -119,7 +119,7 @@ public function onKernelTerminate(PostResponseEvent $event)
     public static function getSubscribedEvents()
     {
         return [
-            KernelEvents::RESPONSE => ['onKernelResponse', -100],
+            KernelEvents::RESPONSE => ['onKernelResponse', -1023],
```

Comment:
1012 be between 1000 and 1024 and still to allow for some room for hooking before or after?

Reply:
updated.